packages/webhooks/test/commands/webhooks/add.test.ts

```diff
@@ -84,7 +84,8 @@ describe('webhooks:add', () => {
     'http://foobar.com',
   ])
   .it('adds a specific pipeline webhook', ctx => {
-    expect(ctx.stdout).to.equal('=== Webhooks Signing Secret\n1234\n')
-    expect(ctx.stderr).to.contain('Adding webhook to example-pipeline... done\n')
+    expect(ctx.stdout).to.contain('=== Webhooks Signing Secret')
+    expect(ctx.stdout).to.contain('1234')
```

Comment:
Split these lines up because the tests don't handle \n correctly with the oclif upgrade